src/layouts/Card.astro

```diff
@@ -0,0 +1,7 @@
+---
+
+---
+
+<div class="mb-5 dark:bg-[#2d272e] px-5 py-7 dark:text-[#f8fafc] dark:border-[#1c1917] border-8 rounded-2xl">
+  <slot />
+</div>
```

Comment:
👋 this is a component not a layout, layouts have like `<html/>` and typically wrap a page.